docs/nidigital/examples.rst

```diff
@@ -3,6 +3,15 @@ Examples
 
 `You can download all nidigital examples for latest version here <https://github.com/ni/nimi-python/releases/download/1.3.2/nidigital_examples.zip>`_
 
+nidigital_burst_with_start_trigger.py
+-------------------------------------
+
+.. literalinclude:: ../../src/nidigital/examples/nidigital_burst_with_start_trigger.py
```

Comment:
This doesn't match the location of the file, right?
`src/nidigital/examples/burst_with_start_trigger/nidigital_burst_with_start_trigger.py`

Reply:
Yeah, you are right. examples.rst does not get regenerated unless a "tox -e clean" is performed. Fixed the generation mako to recursively look through all the folders under examples.